static/index.js

```diff
@@ -103,6 +103,19 @@ var setupCoffeeCache = function(cacheDir) {
   CoffeeCache.register();
 }
 
+var isPortableMode = function() {
+  // No portable mode on non-Windows
+  if (process.platform !== 'win32') return false;
+  
+  // DevMode? Nope
+  var devMode = loadSettings.devMode || !loadSettings.resourcePath.startsWith(process.resourcesPath + path.sep);
```

Comment:
`loadSettings` might be null, see `isRunningCoreSpecs` for an example of guards typically around it.